ql/src/java/org/apache/hadoop/hive/ql/parse/ExplainSemanticAnalyzer.java

```diff
@@ -282,4 +282,11 @@ public boolean skipAuthorization() {
     }
     return super.skipAuthorization();
   }
+
```

Comment:
Does anything in the EXPLAIN output show if it is using cached metadata or not? If so, could we write a Q file test that ensures this continues to work?

Reply:
Looking at the history of "HIVE_OPTIMIZE_HMS_QUERY_CACHE_ENABLED", I am not sure if there is a way to detect currently if a query's metadata is being served from the cache besides looking at the log file (and I would consider this outside of the scope of this PR but others might have differing opinion), so it likely would be difficult to do this via a qtest